repo2docker/app.py

```diff
@@ -99,6 +100,7 @@ def _default_log_level(self):
             CondaBuildPack,
             PipfileBuildPack,
             PythonBuildPack,
+            GuixBuildPack,
```

Comment:
I think the ordering here matters - repo2docker will go from top  to bottom, stopping at the first one. For example, by being below `CondaBuildPack`, if any `environment.yml` file is found in the git repo, `GuixBuildPack` is never called. Let's move this up, just behind the Nix one maybe?

Reply:
Right, done.